packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/quanticHeadlessLoader.js

```diff
@@ -267,7 +267,7 @@ function destroyEngine(engineId) {
  * Register a facet in the store.
  * @param {string} engineId The engine ID.
  * @param {string} facetType
- * @param {{ label: string; facetId: string; format?: function}} data
```

Comment:
This seems more or less like a weird workaround no?

Reply:
This was the only workaround I found to be able to create the exact duplicates of each facet, any ideas for a better alternative solution?